client/post-editor/editor-featured-image/index.jsx

```diff
@@ -121,8 +124,10 @@ class EditorFeaturedImage extends Component {
 	render() {
 		const { site, post } = this.props;
 		const featuredImageId = getFeaturedImageId( post );
+		const dropZoneIsActive = true;  // todo not sure best way to determine if it's active w/ out coupling. maybe create a selector to pick isDraggingOverDocument from the redux store?
```

Comment:
I think you can leverage the already [existing selector](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/is-drop-zone-visible.js#L6).